ansible/roles/common/tasks/readinesscheck.yml

```diff
@@ -2,9 +2,8 @@
 # Wait for pods to be running
 
 - name: wait for pods running
-  ansible.builtin.shell: oc get po -A --no-headers | grep Running | wc -l
+  ansible.builtin.shell: oc get pods -A -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}' | grep -c False
```

Comment:
Just as a suggestion, did you try `oc wait --for=condition=Ready pods --all --all-namespaces`? You would get rid of the ansible loop.

Reply:
I tried this. Unfortunately, it seems to return true before all containers of a given pod are in Ready state.